docs/examples/en/loaders/GLTFLoader.html

```diff
@@ -30,8 +30,10 @@ <h2>Extensions</h2>
 		<ul>
 			<li>KHR_draco_mesh_compression</li>
 			<li>KHR_materials_clearcoat</li>
+			<li>KHR_materials_ior</li>
 			<li>KHR_materials_pbrSpecularGlossiness</li>
 			<li>KHR_materials_unlit</li>
+			<li>KHR_materials_volume</li>
```

Comment:
In https://github.com/mrdoob/three.js/pull/19690, I didn't add `KHR_materials_transmission` to this list because I didn't consider the implementation sufficiently complete to be considered supported. With your work in https://github.com/mrdoob/three.js/pull/21884 I think we could certainly include it now. 👍

Reply:
Oh, I didn't notice that `KHR_materials_transmission` is not in the list. Added, thanks!